backend/models/hashtag.py

```
@@ -0,0 +1,21 @@
+from sqlalchemy import Column, Integer, String, Table, ForeignKey
+from sqlalchemy.dialects.postgresql import ARRAY
```

Ruff (0.8.2)

`sqlalchemy.dialects.postgresql.ARRAY` imported but unused

Remove unused import: `sqlalchemy.dialects.postgresql.ARRAY`

(F401)